test.py

```diff
@@ -1,4 +1,5 @@
 from test1 import func
```

Comment:
The script imports `func` from `test1.py`, but the tests do not actually *use* the imported function.

```suggestion
result = func() ; assert result == expected_result
```